.reuse/dep5

```diff
@@ -29,6 +29,11 @@ Copyright: 2019 Adfinis SyGroup AG <info@adfinis-sygroup.ch>
 License: CC0-1.0
 
 # Source code of caluma
-Files: caluma/*.py docs/*
+Files: caluma/*.py
 Copyright: 2019 Adfinis SyGroup AG <info@adfinis-sygroup.ch>
 License: GPL-3.0-or-later
+
+# Documentation of caluma
+Files: docs/*
```

Comment:
Needs to include the few `*.md` files in the toplevel folder (`CONTRIBUTING.md`, `README.md`)

Reply:
Added these two. Thanks for reviewing.